### PR TITLE
Show dataset size

### PR DESCRIPTION
**Pull request by gheinrich**

### CODE REVIEW

**Comment by lukeyeager** [comment]
Cool stuff, thanks! Any reason we can't add this for ModelJobs, too? Those snapshots can really rack up in disk space.

And it would probably be good to at least write a basic sanity test that checks the size of an empty directory or something.

**Comment by gheinrich** [comment]
Sure, I'll do that!

File: digits/utils/filesystem.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    return human-readable size
+    """
+    for unit in ['','K','M','G','T','P','E','Z']:
+        if abs(num) < 1024.0:
+            if unit != '':
+                return "%3.1f%s%s" % (num, unit, suffix)
+            else:
+                return "%d%s" % (num, suffix)
+        num /= 1024.0
+    return "%.1f%s%s" % (num, 'Y', suffix)
```
Comment (lukeyeager) [inline digits/utils/filesystem.py]: Is this just a refactoring of `digits.utils.sizeof_fmt` ([here](https://github.com/NVIDIA/DIGITS/blob/v2.1.0/digits/utils/__init__.py#L100-L106))?

**Comment by gheinrich** [comment]
Sorry I hadn't noticed there was already a function for this. I'll remove the one I added.

**Comment by lukeyeager** [comment]
Awesome thanks! Go ahead and merge if this is ready - it looks good to me.

**Pull request merged**